cli/crates/production-server/src/server/graph_fetch_method.rs

```diff
@@ -62,6 +63,14 @@ impl GraphFetchMethod {
                 });
             }
             GraphFetchMethod::FromLocal { federated_schema } => {
+                tracing::event!(
```

Comment:
Should we be using tracing events for stdout? In the regular dev server we use `println!()`, if I recall correctly.